src/uct/ib/ud/base/ud_ep.c

```diff
@@ -12,6 +12,10 @@
 #include <uct/ib/base/ib_verbs.h>
 #include <ucs/debug/memtrack.h>
 #include <ucs/debug/log.h>
+#include <ucs/time/time.h>
+
+
+#define UCT_UD_SLOW_TIMER_MAX_TICK(_iface)  ((_iface)->config.peer_timeout / 3)
```

Comment:
why is it / 3?

Reply:
Tick should be less than peer_timeout to avoid false positive errors, `peer_timeout / 2` may be too small for testing on slow hosts or under valgring. In real cases when peer_timeout is a few mins, `peer_timeout / 3` should not be too small.

Reply:
can u pls add comment in the code?